### ISSUE TITLE
Issue with Tattoo's & Fades

### ISSUE THREAD

**Issue by WigglesPuff** [wasabirobby/fivem-appearance#66]
I am Currently on the latest Version of both ESX and Wasabi's Version of five-appearance

The Main-Problem is as follows in steps:

- User Creates New Character
- Fully Configs Their Character(does not show any tattoos but shows the fade of the hair style they picked)
- Spawns At Default Location
- Character appears with tattoos and fade of primary character( this also causes issues with  users on my server who play as animals since those animals come in with Human Ped Fades aka having a cow running around with a fade texture on its head)

Would Greatly Welcome any solution that resolves this issue so users are able to rp with out RP breaking Problems

**Comment by FBFezz** [comment]
- What version of esx are you using
- Are you able to show pictures / a video as an example
- What multi character script are you using?

**Comment by WigglesPuff** [comment]
- **Version:** 1.9.4
- **Multi-Character Script:** Esx-Multicharacter
- https://www.dropbox.com/s/qh14ocbl04he7vh/2023-04-17%2012-07-11.mkv (sorry i only use this cause of the built in player for video) its at the end of the video where it transfers the tattoo and fade of a primary character

**Note:**
i want to also make this note another branch of fivem-appearance(illenium-appearance) experienced this issue and has since already resolved this problem by implementing a fix  at lines 215 to 224 in that branches client.lua of the client folder which resolves the transferring of fades/or tattoos to new characters including the animal peds. just wanted to point this out so you all could reference it and maybe attempt a fix similar to that github repo. the only reason i have not migrated to that version is it doesnt allow full disabling of the cost for shops( aka putting false instead of 0 so you dont have 0$ popping up as cost, unlike wasabi's version that completely disables it), 

Again Thank you for responding to my issues ticket

Sincerely Wiggly

**Comment by FBFezz** [comment]
@wasabirobby Might be something to take a look at if someone has resolved it in another branch

I assume this is the fix your on about

```
function SetInitialClothes(initial)
    client.setPlayerModel(initial.Model)
    -- Fix for tattoo's appearing when creating a new character
    local ped = cache.ped
    client.setPedTattoos(ped, {})
    client.setPedComponents(ped, initial.Components)
    client.setPedProps(ped, initial.Props)
    client.setPedHair(ped, initial.Hair, {})
    ClearPedDecorations(ped)
end
```

**Comment by WigglesPuff** [comment]
@FBFezz Yes, there's probably more then just that key reference in there to the fix, but yes that's the reference basically i was pointing out. also sorry if me just pointing out another branch of fivem-appearance was a no no or caused any issues(ill go back and  remove that reference if you want me too). not my intent just wanted to point something out and give you all a reference for a possible fix to wasabi's version

**Comment by Mrwimz** [comment]
Do we have any updates or solutions for this? Might just be me but i haven't seen another solution anywhere.
